types/amap-js-api/amap-js-api-tests.ts

```diff
@@ -1368,7 +1368,7 @@ testLabelsLayer.on('click', (event: AMap.LabelsLayer.EventMap['click']) => {
             // $ExpectType string | undefined
             textOptions.content;
             if (textOptions.direction !== undefined) {
-                // $ExpectType TextDirection
+                // $ExpectType TextDirection || "top" | "right" | "bottom" | "left" | "center"
```

Comment:
note that because of microsoft/typescript#42304, a few type aliases actually get de-aliased *more*. I'm making this change now because we're not sure whether it's easily fixable.